include/nbl/asset/ICPUBuffer.h

```diff
@@ -27,10 +27,9 @@ class ICPUBuffer final : public asset::IBuffer, public IPreHashed
     public:
         struct SCreationParams : asset::IBuffer::SCreationParams
         {
-            size_t size; // WHY IS THIS SHADOWING THE `asset::IBuffer::SCreationParams` !?
             void* data = nullptr;
-            size_t alignment = _NBL_SIMD_ALIGNMENT;
             core::smart_refctd_ptr<core::refctd_memory_resource> memoryResource = nullptr;
+            size_t alignment = _NBL_SIMD_ALIGNMENT;

```

Comment:
why did you move the alignment parameter so far?

Reply:
in the whole project, no construction overwrites `alignment` but we have a few overwriting `memoryResource`.
```c++
ICPUBuffer::create({ { size }, data, memoryResource});
// over
ICPUBuffer::create({ { size }, data, _NBL_SIMD_ALIGNMENT, memoryResource});
```

Reply:
ok makes sense.